### DISCUSSION TITLE
Just wondering what sort of timeline there might be for the full release of 2.0

### DISCUSSION THREAD

**Discussion by glomotion** [WalletConnect/walletconnect-monorepo#945] [Developer Support]
We're beginning our WalletConnect implementation at the moment, and were wondering how far down the road the release of 2.0 might be? 

Given that there will need to be work done to migrate from v1.x, we're just trying to work out how close to release 2.0 is. If its not far away, we might base our implementation on v2, not v1. 🤔 

Can any body help shed any light on this?

**Comment by chadyj** [comment]
Great to hear you are beginning implementation!

For wallets, we suggest you implement both v1 and v2.  See here for more info https://docs.walletconnect.com/2.0/migration
Several wallets have v2 in production already, but it is in beta so you may prefer to wait until we ship the release candidate in the coming weeks.

For dapps, you should use v1, since v2 requires that wallets migrate first (see above link)

**Reply by RivarolaNicolas** [reply]
Hey @chadyj do you have an estimate of when the 2.0rc will be released? The updated documentation will be released alongside the RC or it could be released before or after?

Thank you!

**Reply by chadyj** [reply]
No specific date, but we are aiming soon-ish :) 
Documentation will be updated at the same time.
